src/client/dfuse/dfuse_core.c

```diff
@@ -885,6 +889,47 @@ dfuse_ie_close(struct dfuse_projection_info *fs_handle,
 	D_FREE(ie);
 }
 
+#if HAVE_FUSE_LOG
+static void
+dfuse_fuse_log(enum fuse_log_level level,
+	const char *fmt,
+	va_list ap)
+{
+	int mask;
+	int d_level = DLOG_EMIT;
+
+	switch (level) {
+	case FUSE_LOG_EMERG:
+		d_level = DLOG_EMERG;
+		break;
+	case FUSE_LOG_ALERT:
+		d_level = DLOG_ALERT;
+		break;
+	case FUSE_LOG_CRIT:
+		d_level = DLOG_CRIT;
+		break;
+	case FUSE_LOG_ERR:
+		d_level = DLOG_ERR;
+		break;
+	case FUSE_LOG_WARNING:
+		d_level = DLOG_WARN;
+		break;
+	case FUSE_LOG_NOTICE:
+		d_level = DLOG_NOTE;
+		break;
+	case FUSE_LOG_INFO:
+		d_level = DLOG_INFO;
+		break;
+	case FUSE_LOG_DEBUG:
+		d_level = DLOG_DBG;
```

Comment:
I think you just want DB_IO here.   DLOG_DBG is a bit mask that covers all debug bits so this is equivalent to something like

0xff | 0x1 where DLOG_DBG is 0xff and DB_IO is 0x1 (not actual values).